dotcom-rendering/scripts/deploy/riff-raff.yaml

```diff
@@ -66,7 +66,26 @@ deployments:
     dependencies:
       - frontend-static
       - article-rendering-cfn
+  # facia-rendering cloudformation
+  facia-rendering-cfn:
+    template: cloudformation
+    parameters:
+      templateStagePaths:
+        CODE: FaciaRendering-CODE.template.json
+        PROD: FaciaRendering-PROD.template.json
+      cloudFormationStackByTags: false
+      cloudFormationStackName: facia-rendering
+      amiParameter: AMIFaciarendering
```

Comment:
Non-blocking: Is it possible to camelcase this param to `AMIFaciaRendering` so its slightly easier to read?